scalableminds/webknossos
# fix volume mode and trigger update action when creating a new cell

This PR fixes the volume mode that was broken by the webpack3 PR earlier and also properly triggers update actions in volume mode (when creating a cell, activating a cell).

### Steps to test:
- Open volume mode and create multiple cells using "c" without moving. The save icon should switch to unsaved and waiting or triggering a save should lead to an updated activeCellId once the page is refreshed.

### Issues:
- fixes #1889

------
